src/dipdup/yaml.py

```diff
@@ -121,3 +124,7 @@ def dump(self) -> str:
         buffer = StringIO()
         yaml.dump(config_yaml, buffer)
         return buffer.getvalue()
+    
+    def validate_version(self) -> None:
+        if self['spec_version'] != __spec_version__:
+            raise ConfigurationError(f"Inappropriate configuration version expected {__spec_version__} got {self['spec_version']} check https://docs.dipdup.io/config/spec_version")
```

Comment:
Possibly better wording:

> Unknown spec version {version}; see https://docs.dipdup.io/config/spec_version

Or maybe a separate exception to make help message multiline?